tests/Rector/ClassMethod/ReplaceTestAnnotationWithPrefixedFunctionRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\ClassMethod\ReplaceTestAnnotationWithPrefixedFunctionRector\Fixture;
+
+class SomeTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @test
+     */
+    public function onePlusOneShouldBeTwo()
+    {
+        $this->assertSame(2, 1+1);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\ClassMethod\ReplaceTestAnnotationWithPrefixedFunctionRector\Fixture;
+
+class SomeTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @test
+     */
+    public function testOnePlusOneShouldBeTwo()
+    {
+        $this->assertSame(2, 1+1);
+    }
```

Comment:
Thanks for the rule :+1: 

It should remove the annotaiton too, as it's not needed anymore thanks to "test" prefix in method name:

```suggestion
    public function testOnePlusOneShouldBeTwo()
    {
        $this->assertSame(2, 1+1);
    }
```

Reply:
I wasn't sure if we should do it in the same rector or use a different one so I went with "simplest" first. I agree that we should probably have it in one rule :)